decidim-dipgra_census_authorization/decidim-dipgra_census_authorization.gemspec

```diff
@@ -26,4 +26,5 @@ Gem::Specification.new do |s|
 
   s.add_development_dependency "decidim-dev", DECIDIM_VERSION
   s.add_development_dependency "faker"
+  s.metadata["rubygems_mfa_required"] = "true"
```

Comment:
what is this for?

Reply:
Rubocop: C: [Correctable] Gemspec/RequireMFA: metadata['rubygems_mfa_required'] must be set to 'true'.